integration-cli/requirements_unix.go

```diff
@@ -135,6 +136,22 @@ var (
 		},
 		"Test cannot be run wihtout suppport for ovelayfs",
```

Comment:
Oh (no need to change, I'm just seeing them now); `s/wihtout/without/` and `s/ovelayfs/overlayfs/`